src/request_queue.js

```diff
@@ -550,9 +550,39 @@ export class RequestQueue {
      * @return {Promise<number>}
      */
     async handledCount() {
+        // TODO: We should deprectate this function in favor of getInfo()
         const queueInfo = await requestQueues.getQueue({ queueId: this.queueId });
         return queueInfo.handledRequestCount;
     }
+
+    /**
+     * Returns an object containing general information about the request queue.
+     *
+     * **Example:**
+     * ```
+     * {
+     *   "id": "WkzbQMuFYuamGv3YF",
+     *   "name": "my-queue",
+     *   "userId": "wRsJZtadYvn4mBZmm",
+     *   "createdAt": new Date("2015-12-12T07:34:14.202Z"),
+     *   "modifiedAt": new Date("2015-12-13T08:36:13.202Z"),
+     *   "accessedAt": new Date("2015-12-14T08:36:13.202Z"),
+     *   totalRequestCount: 0,
+     *   handledRequestCount: 0,
+     *   pendingRequestCount: 0,
+     * }
+     * ```
+     * @param {Object} [options={}]
+     *   Additional options passed to Apify API Client's
+     *   [getQueue](https://www.apify.com/docs/api/apify-client-js/latest#ApifyClient-requestQueues-getQueue) function.
+     * @returns {Promise<Object>}
+     */
+    getInfo(options = {}) {
```

Comment:
Also I think that we can skip options parameter here

Reply:
Done.